src/server/config/schema.js

```diff
@@ -143,10 +143,20 @@ module.exports = () => Joi.object({
     }),
     profile: Joi.boolean().default(false)
   }).default(),
-
   status: Joi.object({
     allowAnonymous: Joi.boolean().default(false)
   }).default(),
+  vectormap: Joi.object({
```

Comment:
This config section snuck in here, even though we won't be adding vectormap support with this PR.